inputstream.adaptive/resources/language/resource.language.nl_nl/strings.po

```diff
@@ -0,0 +1,105 @@
+# Kodi Media Center language file
+# Addon Name: Inputstream.adaptive
+# Addon id: inputstream.adaptive
+# Addon Provider: peak3d
+
+msgid ""
+msgstr ""
+
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
```

Comment:
"Language: nl_NL\n"

Reply:
Ik heb er `"language: nl\n"` van gemaakt, in dezelfde lijn als het was.